app/views/passengers/show.haml

```diff
@@ -16,31 +16,19 @@
   = @passenger.status
 %p
   %b Registered with disability services?
-  - if @passenger.registered_with_disability_services?
-    Yes
-  - else
-    No
+  %td.yes= yes_no_image @passenger.registered_with_disability_services?
```

Comment:
Great job with this! I know you had some trouble to begin with, but you got there, @Chauce!